core/src/main/scripts/importer/validateData.py

```diff
@@ -1258,6 +1258,14 @@ def skipValidation(self, data):
         """Test whether the mutation is silent and should be skipped."""
         is_silent = False
         variant_classification = data[self.cols.index('Variant_Classification')]
+        if 'variant_classification_filter' in self.meta_dict:
+            self.SKIP_VARIANT_TYPES = [x.strip() 
+                                       for x 
+                                       in self.meta_dict['variant_classification_filter'].split(',')]
+        
+        # check that the values written by the user are valid values, otherwise raise a warning
+        for value in self.SKIP_VARIANT_TYPES:    
+            self.checkVariantClassification(value)
```

Comment:
I would not give a warning here, since these values will be skipped/ filtered. The warning given here is anyway not accurate for this scenario.